src/main/java/com/softserve/academy/spaced/repetition/controller/CardRatingController.java

```diff
@@ -32,35 +31,31 @@
      * @param cardId
      * @return CardRatingDTO
      */
-    @GetMapping("api/rate/card/{cardId}")
-    public ResponseEntity<CardRatingPublicDTO> getCardRatingById(@PathVariable Long cardId) {
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping
+    public CardRatingPublicDTO getCardRatingById(@PathVariable Long cardId) {
         CardRating cardRating = cardRatingService.getCardRatingById(cardId);
         Link selfLink = linkTo(methodOn(CardRatingController.class)
                 .getCardRatingById(cardRating.getId())).withRel("cardRating");
```

Comment:
We should use `withSelfRel`, if we want to build link to indicate current resource.

Reply:
this controllers never used, so i did`t pay attention  for this controllers